code/game/jobs/job/command/auxiliary/senior.dm

```diff
@@ -1,3 +1,5 @@
+#define SEA_RANKS list("ME7", "ME8", "ME8E", "ME9", "ME9E")
```

Comment:
```suggestion
#define SEA_RANKS list("Gunnery Sergeant" = "ME7", "Master Sergeant" = "ME8", "First Sergeant" = "ME8E", "Master Gunnery Sergeant" = "ME9", "Sergeant Major" = "ME9E")
```
This will probably be a bit more human readable, assuming I am not about to make a dozen mistakes in this suggestion...